<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>S R B | Inicio de Sesi칩n</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../public/img/icon-SRB.png</normaloff>../../public/img/icon-SRB.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(139, 191, 255);</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="lb_image">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>431</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../public/img/logo-SRB.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_access">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Verdana&quot;;
background-color: rgb(0,0,127);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Iniciar Sesi칩n</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="horizontalFrame_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>281</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(240,240,240);
border-radius: 20px;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="lb_pass">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../public/img/candado.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_password">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Utendo&quot;;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>contrase침a</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="horizontalFrame">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>281</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(240,240,240);
border-radius: 20px;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="lb_user">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../public/img/usuario.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_user">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Utendo</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Utendo&quot;;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>usuario o email</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lb_error">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>410</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(184, 0, 0);
font: italic 10pt &quot;Verdana&quot;;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>291</width>
     <height>73</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 30pt &quot;Utendo&quot;;
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Inicio de Sesi칩n</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
